lms/djangoapps/courseware/masquerade.py

```diff
@@ -128,6 +157,7 @@ def get(self, request, course_key_string):
                     }
                     for group in partition.groups
                 ])
+        data['active']['name'] = CourseMasquerade.get_active_group_name(data)
```

Comment:
This way we can use it to display in the status alert.

Note: We _could_ parse this out on the frontend, assuming we have access to `data['active']` and `data['available']`... :shrug: